deployment/modules/space/main.tf

```diff
@@ -9,9 +9,9 @@ resource "cloudfoundry_user_provided_service" "login-gov" {
   name  = "login-gov"
   space = data.cloudfoundry_space.space.id
   credentials = {
-    certificate = lookup(var.cf_strapi_logingov_cert, var.cf_env)
+    certificate = var.cf_strapi_logingov_cert
     public_key  = var.cf_strapi_logingov_key
-    issuer      = lookup(var.cf_strapi_logingov_issuer, var.cf_env)
+    issuer      = var.cf_strapi_logingov_issuer
```

Comment:
I did this because I was hoping we could store the secrets in a Cloud Foundry user defined service